baseline.gradle

```diff
@@ -76,6 +76,7 @@ subprojects {
           '-Xep:EqualsGetClass:OFF',
           // patterns that are allowed
           '-Xep:MissingCasesInEnumSwitch:OFF',
+          '-Xep:ShutdownHook:OFF',
```

Comment:
Instead of adding this, is it possible to add an annotation suppressing the error in just that one part of the code?

If not, then that's ok.

Reply:
Thanks. I've fixed it.